src/wasm.h

```diff
@@ -86,6 +86,10 @@ struct Name : public cashew::IString {
   static Name fromInt(size_t i) {
     return cashew::IString(std::to_string(i).c_str(), false);
   }
+
+  bool hasSubstring(cashew::IString substring) {
```

Comment:
I'd rather not do this in `wasm.h` since it tries to be super-minimal. And it seems like there are dozens of other small functions like that.

Perhaps we should move `Name` out of `wasm.h`, into `name.h` perhaps? Or `support/name.h`? In that case, having it in the class sounds ok to me.